### ISSUE TITLE
500s for bots visiting spending breakdown

### ISSUE THREAD

**Issue by sebbacon** [ebmdatalab/openprescribing#1312]
[new relic](https://rpm.newrelic.com/accounts/1550585/applications/45170403/filterable_errors#/show/9d1b3bbf-12b2-11e9-9667-0242ac11000d_0_4517/stack_trace?top_facet=transactionUiName&primary_facet=error.class&barchart=barchart)
 (also in Sentry)

```
Stack trace
exceptions:ValueError: max() arg is an empty sequence
Traceback (most recent call last):
File "/webapps/openprescribing/.venv/bin/gunicorn", line 11, in <module>
File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 61, in run
File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 223, in run
File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 72, in run
File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 212, in run
File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 545, in manage_workers
File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 616, in spawn_workers
File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/gunicorn/workers/base.py", line 134, in init_process
File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/gunicorn/workers/sync.py", line 124, in run
File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/gunicorn/workers/sync.py", line 68, in run_for_one
File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/gunicorn/workers/sync.py", line 30, in accept
File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/gunicorn/workers/sync.py", line 135, in handle
File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/gunicorn/workers/sync.py", line 176, in handle_request
File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/newrelic/api/web_transaction.py", line 1291, in _nr_wsgi_application_wrapper_
File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/newrelic/api/web_transaction.py", line 818, in __init__
File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/newrelic/api/web_transaction.py", line 1180, in _nr_wsgi_application_wrapper_
File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/django/core/handlers/wsgi.py", line 157, in __call__
File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/newrelic/hooks/framework_django.py", line 423, in _nr_wrapper_BaseHandler_get_response_
File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 124, in get_response
File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/newrelic/hooks/framework_django.py", line 1203, in _wrapper
File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 249, in _legacy_get_response
File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
File "/webapps/openprescribing/.venv/local/lib/python2.7/site-packages/newrelic/hooks/framework_django.py", line 544, in wrapper
File "/webapps/openprescribing/openprescribing/frontend/views/views.py", line 662, in spending_for_one_entityhttps://rpm.newrelic.com/accounts/1550585/applications/45170403/filterable_errors#/show/9d1b3bbf-12b2-11e9-9667-0242ac11000d_0_4517/stack_trace?top_facet=transactionUiName&primary_facet=error.class&barchart=barchart
```

**Comment by evansd** [comment]
Ah yes, this was the thing I fixed in #1309 but I haven't yet deployed it. I'll do so now.

**Comment by sebbacon** [comment]
:+1:
